Source/Moq.Analyzers.Test/Data/AbstractClass.Bad.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Moq.Analyzers.Test.Data
+{
+    internal class MyBadUnitTests
+    {
+        private void TestBad()
```

Comment:
From conversation, some test cases missing:

- Pass an argument to a constructor that takes no arguments
- Pass (string, int) to a constructor that takes (int, string) or equivalent test
- Pass the incorrect number of arguments to constructor